### PR TITLE
Updated the Domain regex

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-14916)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Ni-Knight** [comment]
There are no links to the JIRA issues.

**Comment by MLainer1** [comment]
> There are no links to the JIRA issues.

Updated the link

**Review by Ni-Knight — APPROVED** [review]

File: Packs/CommonTypes/ReleaseNotes/3_3_28.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Indicator Types
+- **domainRepUnified**
+- Updated the Domain regex
```
Comment (dantavori) [inline Packs/CommonTypes/ReleaseNotes/3_3_28.md:4]:
```suggestion
- Updated the Domain regex.
```

File: Packs/CommonScripts/Scripts/ExtractDomainAndFQDNFromUrlAndEmail/ExtractDomainAndFQDNFromUrlAndEmail.py [hunk 2]
```diff
@@ -66,14 +67,32 @@ def get_fqdn(the_input):
     return fqdn
 
 
-def extract_fqdn(the_input):
+def pre_process_input(the_input):
+    the_input = the_input.removesuffix('.')
+    the_input = the_input.removeprefix('/')
```
Comment (dantavori) [inline Packs/CommonScripts/Scripts/ExtractDomainAndFQDNFromUrlAndEmail/ExtractDomainAndFQDNFromUrlAndEmail.py:72]: as discussed, need to check if this suffices.

File: Packs/CommonTypes/ReleaseNotes/3_3_28.md (hunk 1 above)
Comment (dantavori) [reply Packs/CommonTypes/ReleaseNotes/3_3_28.md:4]: what about the change in the formatting script?

**Pull request merged**